File: plot/src/main/java/net/fwitz/math/main/bifurc/BifurcLogisticMap.java
package net.fwitz.math.main.bifurc;

import net.fwitz.math.fractal.bifurc.BifurcLogisticMapFunction;
import net.fwitz.math.plot.bifurc.LogisticMapPlot;

public class BifurcLogisticMap {
    public static void main(String[] args) {
        new LogisticMapPlot("rx(1-x)")
                .computeFn(BifurcLogisticMapFunction::evaluate)
                .render();
    }
}

File: main/src/main/kotlin/net/fwitz/math/main/bifurc/BifurcLogisticMap.kt
package net.fwitz.math.main.bifurc

import net.fwitz.math.fractal.bifurc.BifurcLogisticMapFunction
import net.fwitz.math.plot.bifurc.LogisticMapPlot

object BifurcLogisticMap {
    @JvmStatic
    fun main(args: Array<String>) = LogisticMapPlot("rx(1-x)")
        .computeFn { x -> BifurcLogisticMapFunction.evaluate(x) }
        .render()
}